doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -19,12 +19,14 @@ endif::[]
 
 To create a {PRODUCT} clustered development environment, you must perform the following tasks:
 
-* Configure {EAP} {EAP_VERSION} with {DATAGRID} {DATAGRID_VERSION} on a machine.
-* Configure AMQ Broker, a Java messaging server (JMS) broker, on a machine.
-* Configure an NFS file server on a machine.
-* Download {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}, then install them on each machine that is to become a cluster node.
-* Configure and start {CENTRAL} on each cluster node to start the operation of the cluster.
-
+. Configure the following components on a system that will be a node of the cluster:
+//Is this correct? Is there a 'main' system with clustering,or a controller? What is the correct term? If so, are these the components that must be installed?
```

Comment:
I think it is ok. It is basically possible to configure everything on a single machine and also one item per machine but it is up to customers. I would say there is nothing like main node.

Reply:
OK, I removed the comment. Thanks.